bigframes/ml/compose.py

```diff
@@ -46,6 +46,93 @@
 )
 
 
+class SQLScalarColumnTransformer:
+    """
+    Wrapper for plain SQL code contained in a ColumnTransformer.
+
+    Create a single column transformer in plain sql.
+    This transformer can only be used inside ColumnTransformer.
+
+    When creating an instance '{0}' can be used as placeholder
+    for the column to transform:
+
+        SQLScalarColumnTransformer("{0}+1")
+
+    The default target column gets the prefix 'transformed\_'
```

Comment:
Looks like the linter is complaining about this one:

```suggestion
    The default target column gets the prefix 'transformed_'
```

Alternatively, maybe switching to an `r"""` above will fix that?